controls/scheduler/built-in-dialogs.md

```diff
@@ -0,0 +1,114 @@
+
+#### Edit Recurrence Dialog
+
+Lets the end user configure the recurrence settings of an appointment.
+
+![Telerik .NET MAUI Scheduler EditRecurrenceDialog](images/scheduler-dialogs-editrecurrence.png)
+
+#### Edit Recurrence Choice Dialog
+
+When the end user interacts with a recurring appointment, lets them choose if they want to edit a single occurrence or the entire series.
+
+![Telerik .NET MAUI Scheduler EditRecurrenceChoiceDialog](images/scheduler-dialogs-recurrencechoice.png)
+
+#### Delete Recurrence Choice Dialog
+
+When the end user interacts with a recurring appointment, lets them choose if they want to delete a single occurrence or the entire series.
+
+![Telerik .NET MAUI Scheduler DeleteRecurrenceChoiceDialog](images/scheduler-dialogs-deleterecurrencechoice.png)
+
+#### Delete Appointment Choice Dialog
+
+(Desktop-only) When the end user attempts to delete an appointment, asks for confirmation.
+
+![Telerik .NET MAUI Scheduler DeleteAppointmentChoiceDialog](images/scheduler-dialogs-deleteappointmentchoice.png)
+
+## Methods
+
+Use the following methods to show the built-in dialogs of the Scheduler:
+
+* `CreateAppointmentWithDialog(DataRange range)`&mdash;Opens a dialog for creating an appointment.
+* `EditAppointmentWithDialog(Occurrence occurrence)`&mdash;Opens a dialog for editing an exsisting appointment.
+* `DeleteAppointmentWithDialog(Occurrence occurrence)`&mdash;Opens a dialog for deleting an existing appointment.
+
+The following example demonstrates how to use the methods for managing the built-in dialogs:
+
+**1.** Create a sample view with a `RadScheduler` instance and a few buttons:
+
+<snippet id='scheduler-dialogs-methods-xaml' />
+
+**2.** Add the buttons' clicked event handlers in the code-behind:
+
+<snippet id='scheduler-dialogs-methods-eventhandlers' />
+
+## Events
+
+The Scheduler exposes the following events for its built-in dialogs:
+
+* `DialogOpening`&mdash;Raised when a dialog is about to be opened. The `DialogOpening` event handler receives two parameters:
+    * The sender argument, which is of type `object`, but can be cast to the `RadScheduler` type.
+    * A `SchedulerDialogOpeningEventArgs` object with the following properties:
+       * `DialogType`&mdash;Provides the dialog type.
+       * `Cancel`&mdash;If `True`, cancels the dialog opening.
+* `DialogClosing`&mdash;Raised when a dialog is about to be closed. The `DialogClosing` event handler receives two parameters:
+    * The sender argument, which is of type `object`, but can be cast to the `RadScheduler` type.
+    * A `SchedulerDialogClosingEventArgs` object with the following properties:
+       * `DialogType`&mdash;Provides the dialog type.
+       * `Cancel`&mdash;If `True`, cancels the dialog closing.
+       * `DialogResult (bool?)`&mdash;Indicates how the dialog was closed. `null` when closed with the **Close** button. `False` when closed with the **Cancel** button. `True` when <BE SPECIFIC>.
```

Comment:
Check the end of the line :) Either provide explicit details or go with "`True` in all other cases."